### PR TITLE
feat(cli): upgrade packages for React and WC

### LINKED ISSUE #1191: Implement `ig upgrade` for WebComponents and React
Implement support for `ig upgrade-packages` command in WebComponents and React platforms.
Currently, only Angular is supported.

### PR DESCRIPTION
**Pull request by igdmdimitrov**
Closes #1191

### CODE REVIEW

File: packages/cli/templates/Update.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	let logicFilesExtension = "";
+	let upgradeable: PackageDefinition[] = [];
+
+	const config = ProjectConfig.getConfig();
+	const framework = config.project.framework;
+	switch (framework.toLowerCase()) {
+		// case "angular":
+		// 	guideLink =
+		// 	  "https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/ignite-ui-licensing";
+		// 	break;
```
Comment (gedinakova) [inline packages/cli/templates/Update.ts:17]: Do we need this commented out chunk?

File: packages/cli/templates/Update.spec.ts [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+import 'igniteui-react-grids/grids/themes/light/bootstrap.css'
+
+IgrGridModule.register();
+
+export default function Home() {
+	const title = 'igniteui-react example';
+}`,
+				expected:
+`import { something } from 'module';
+import { bait } from 'igniteui-react-core';
```
Comment (gedinakova) [inline packages/cli/templates/Update.spec.ts:135]: Shouldn't 'igniteui-react-core' be licensed?

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: packages/cli/templates/Update.spec.ts (hunk 2 above)
Comment (igdmdimitrov) [reply packages/cli/templates/Update.spec.ts:135]: It is not in the MockFile dependencies and that's why it is not licensed. I'll add it to dependencies in order to be licensed after upgrade.

File: packages/cli/templates/Update.ts (hunk 1 above)
Comment (igdmdimitrov) [reply packages/cli/templates/Update.ts:17]: Depends if we want to keep refactoring and support Angular upgrade here in the same file.

**Review by jackofdiamond5 — APPROVED** [review]

**Review by gedinakova — APPROVED** [review]

**Pull request merged**